tool/tsh/tsh.go

```diff
@@ -1259,20 +1264,25 @@ func newTraceProvider(cf *CLIConf, command string, ignored []string) (*tracing.P
 
 // onVersion prints version info.
 func onVersion(cf *CLIConf) error {
-	proxyVersion, err := fetchProxyVersion(cf)
-	if err != nil {
-		fmt.Fprintf(cf.Stderr(), "Failed to fetch proxy version: %s\n", err)
+	proxyVersionCheck := ""
```

Comment:
This variable holds the actual version of the proxy, so proxyVersion seems like a better choice of name. I would just call the inner var pv or something.

Reply:
changed